README.md

```diff
@@ -0,0 +1,65 @@
+# Sensu Go Ansible Collection
+
+Sensu Go [Ansible Collection][collection] contains:
+
+* Ansible role to install [Sensu Go][sensu] on your nodes.
```

Comment:
Strictly speaking: roles, not role, right?  This sentence makes me think we're only referring to the role for installing agents, but there's the backend role as well.

Reply:
I think we inherited this text from the original author and never updated it. Good catch.